A Tutorial On How To Create An Application Using 3Box Comments Plugin.md

```diff
@@ -0,0 +1,22 @@
+##  Introduction to 3Box
+The first part of the article will be a brief intorduction to 3box, 
+the tools it offers for the developers & the technology it uses
+so that the audience is aware that 3box-comment-plugin is 
+just one of the several tools they can use in 3box sdk 
+and at the same time will be knowledgeable about the background of 3box & the technology it uses.
+
+
+## What is 3box-comment-plugin?
+In the second part of the article i will discuss what is 3box-comment-plugin & what it can do.
+I will also provide a sample on how it can empower a simple dapp and turn it into an engaging,
+exciting and sociable decentralized application.
+
+## How to Empower your dapp with 3box-comment-plugin
+The third part of the article will be the step by step proccess on how to integrate 3box-comment plugin on your application.
+this will include the highlighting keywords from 3box documentation, actual code from my application, 
```

Comment:
```suggestion
this will include highlighting keywords from the 3box documentation, and actual code from my application, 
```

What does 
> actual code from my application
mean?

Aren't you showing code step by step?